packages/python-editor/src/index.ts

```diff
@@ -198,23 +199,30 @@ const extension: JupyterFrontEndPlugin<void> = {
     });
 
     /**
-     * Create new python file from launcher and file menu
+     * Create new python editor from launcher and file menu
      */
 
     // Add a python launcher
     if (launcher) {
       launcher.add({
-        command: commandIDs.createNewPythonFile,
+        command: commandIDs.createNewPythonEditor,
         category: 'Elyra',
         rank: 4
       });
     }
 
     if (menu) {
-      // Add new python file creation to the file menu
+      // Add new python editor creation to the file menu
+      // const scriptEditorMenu = new Menu({ commands: app.commands });
+      // scriptEditorMenu.addItem({
+      //   command: commandIDs.createNewPythonEditor,
+      //   args: { isMenu: true }
+      // });
+      // menu.addMenu(scriptEditorMenu, { rank: 92 });
```

Comment:
is this a remanent of an earlier code change? or is it how you implement grouping the editors together like I suggested in a previous comment?

Reply:
This is what I've tried so far for this PR, shouldn't have included this comment a commit yet, whoops.
I was following [this documentation](https://github.com/jupyterlab/jupyterlab/blob/d7f1704307c2e345f247e1a411bdbae50f2a5708/docs/source/extension/extension_points.rst#adding-a-new-menu) for adding new menus, it didn't really work for this purpose.
As done for the pipeline editor (and following [this doc section](https://github.com/jupyterlab/jupyterlab/blob/d7f1704307c2e345f247e1a411bdbae50f2a5708/docs/source/extension/extension_points.rst#adding-a-new-command-to-an-existing-menu)), the grouping requires an array of menu items. For the script editors this is a little tricky as we need to group them in the script editor extension, even though it is agnostic of the subclasses that extend it so it will be called twice, once by each editor, then we need some sort of promise resolution to get the results and group them together at the same time.

Reply:
code that touches the jupyter application like this should be in a library package like script-editor. If that's the only way to group them together like I requested then I would rather leave them grouped separate then